gen.py

```diff
@@ -326,15 +368,22 @@ def output_overview_page(self, standard, page, reference_pages):
             ).encode('utf8'))
 
 
-
     def element_loop(self, element, path):
+        """Find child elements for a given input element.
+
+        Args:
+            element (lxml.etree._Element): The base element to find child elements for.
+            path (str): Unused.
```

Comment:
If it's not used can it be removed?

Reply:
With there being no tests for this function, I'm not comfortable removing parameters!